temporal-sdk/src/main/java/io/temporal/internal/worker/Poller.java

```diff
@@ -213,13 +222,7 @@ public String toString() {
     @Override
     public void run() {
       try {
-        if (pollExecutor.isShutdown()) {
```

Comment:
This is really not needed here. This was just checked by us in finally AND by the thread pool.